From e1e834e87397503e501cd088fbcdbb70509ac2ae Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Wed, 10 May 2023 17:17:42 -0700
Subject: [PATCH 4/6] imx8mp_var_dart: use common
 spl_mmc_get_uboot_raw_sector()

This function moved as part of 8mm to mach-imx/spl.c

Signed-off-by: Michael Scott <mike@foundries.io>
---
 board/variscite/imx8mp_var_dart/imx8mp_var_dart.c | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/board/variscite/imx8mp_var_dart/imx8mp_var_dart.c b/board/variscite/imx8mp_var_dart/imx8mp_var_dart.c
index 6b915dd75a0..29fdb0ef6e2 100644
--- a/board/variscite/imx8mp_var_dart/imx8mp_var_dart.c
+++ b/board/variscite/imx8mp_var_dart/imx8mp_var_dart.c
@@ -417,18 +417,3 @@ bool is_power_key_pressed(void) {
 	return (bool)(!!(readl(SNVS_HPSR) & (0x1 << 6)));
 }
 #endif
-
-#ifdef CONFIG_SPL_MMC
-
-#define UBOOT_RAW_SECTOR_OFFSET 0x40
-unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
-{
-	u32 boot_dev = spl_boot_device();
-	switch (boot_dev) {
-		case BOOT_DEVICE_MMC2:
-			return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR - UBOOT_RAW_SECTOR_OFFSET;
-		default:
-			return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR;
-	}
-}
-#endif
-- 
2.40.1

